src/java.desktop/macosx/classes/sun/font/CStrike.java

```diff
@@ -102,6 +102,17 @@ public long getNativeStrikePtr() {
         final double[] glyphTx = new double[6];
         desc.glyphTx.getMatrix(glyphTx);
 
+        for (int i = 0; i < 6; i++) {
+            if (Double.isFinite(glyphTx[i])) {
+                continue;
+            }
+            for (int j = 0; j < 6; j++) {
+                glyphTx[j] = 0;
+            }
```

Comment:
```suggestion
            Arrays.fill(glyphTx, 0);
```
The intent of resetting the matrix could be clearer this way.